v2_migrations/2024-08-23-112510_payment_methods_v2_db_changes/up.sql

```diff
@@ -0,0 +1,34 @@
+-- Your SQL goes here
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS accepted_currency;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS scheme;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS token;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS cardholder_name;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS issuer_name;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS issuer_country;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS payer_country;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS is_stored;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS direct_debit_token;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS swift_code;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS payment_method_issuer;
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS payment_method_issuer_code;
+
+DROP TYPE IF EXISTS "PaymentMethodIssuerCode";
+
+ALTER TABLE payment_methods DROP CONSTRAINT IF EXISTS payment_methods_pkey;
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS id;
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS id VARCHAR(64) PRIMARY KEY;
```

Comment:
We cannot be adding a primary key column to an existing database though, since the value in the column for all existing records would be `NULL`?

This would have to be done along the lines of:

- Drop existing `id` column.
- Add new `id` column, without a `NOT NULL` or a `PRIMARY KEY` constraint.
- Copy over data from `payment_method_id` column to `id` column.
- Drop primary key constraint on `payment_method_id` column.
- Add primary key constraint on `id` column.
- Drop `payment_method_id` column (eventually).